src/basic/raw-clone.h

```diff
@@ -25,7 +25,7 @@
  * require CLONE_VM) are not usable.
  *
  * Additionally, as this function does not pass the ptid, newtls and ctid parameters to the kernel, flags must not
```

Comment:
Maybe better to update the comment here?

Reply:
Hmm, what precisely? Note that pidfd is stored in the parent_tid param.

Reply:
Then, maybe better to mention that?